editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/properties/FusePropertySection.java

```diff
@@ -522,6 +549,17 @@ protected void dataFormatChanged(String dataformat, Composite eform, AbstractCam
 		eform.layout(true);
 		aTabbedPropertySheetPage.resizeScrolledComposite();
 	}
+	
+	private void updateDependencies(ArrayList<Dependency> dependencies) {
+		if (dependencies != null && !dependencies.isEmpty()) {
```

Comment:
What do you think of putting the checks inside the MavenUtils method itself? We will have a more robust utility method.